### PR TITLE
data: SageKyle

### PR DESCRIPTION
**Pull request by SageKyle**

### CODE REVIEW

File: data/SageKyle.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    { "icon": "FaTwitter", "url": "https://twitter.com/Ominyi_Sage" },
+    { "icon": "FaGithub", "url": "https://github.com/Ominyi_Sage" },
+    { "icon": "FaLinkedin", "url": "https://linkedin.com/paul-ominyi-63229821a" },
+    { "icon": "FaGlobe", "url": "https://saje.hashnode.dev" }
+  ],
+   "links": [
+    {
+      "name": "Read my articles about web development",
+      "url": "https://saje.hashnode.dev",
+      "icon": "FaHashnode",
```
Comment (Panquesito7) [inline data/SageKyle.json:17]: Wrong icon name.
```suggestion
      "icon": "SiHashnode",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: data/SageKyle.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
+{
+  "name": "ominyi Paul, Saje",
+  "displayStatsPublic": true,
```
Comment (kumarsonsoff3) [inline data/SageKyle.json:3]: We recently have removed the public stats feature. So `displayStatsPublic` property is no longer needed. 
Please remove this from your json. 

```suggestion
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: data/SageKyle.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    },
+    {
+      "name": "Let's connect on LinkedIn",
+      "url": "https://linkedin.com/paul-ominyi-63229821a",
+      "icon": "FaLinkedin"
+    },
+     {
+      "name": "View my daily progress on NewDev",
+      "url": "https://newdev.io/paul",
+       "color": "blue",
```
Comment (Panquesito7) [inline data/SageKyle.json:38]: Wrong indentation.
```suggestion
      "color": "blue",
```

**Comment by SageKyle** [comment]
I used a JSON formatter to update the format of the entire file.

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**